pyproject.toml

```diff
@@ -53,7 +56,7 @@ pytest-recording = "^0.12.0"
 types-requests = "^2.26"
 
 [tool.poetry.extras]
-linting = ["black", "mypy", "prospector"]
+linting = ["mypy", "prospector"]
```

Comment:
Why this change? Shouldn't we keep black in the extras, or is that covered in the precommit?

Reply:
My thought was that since we install black via pre-commit there is no reason to install it in the environment, too. This way we don't need to reference `black` in multiple files (`pyproject.toml` vs `.pre-commit-config.yaml`) and keep versions in sync. So, since `black` was no longer referenced in `pyproject.toml` I removed it from the extras, too.

Reply:
PS. You probably already know this, but just in case, black has introduced a forward-compatible policy:
> Starting in 2022, the formatting output will be stable for the releases made in the same year (other than unintentional bugs). 
Therefore versions `2022.3` up to `2022.12` should only contain bugfixes. Breaking changes will come in `2023.*`
[source](https://black.readthedocs.io/en/stable/faq.html#how-stable-is-black-s-style)

Reply:
I didn't know that, that's great! That will make things a lot easier with pinning versioning